spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -587,17 +609,4 @@ def verify_session_cookie
       end
     end
   end
-
-  context 'when not logged in' do
-    describe 'POST saml_callback' do
-      context 'loa3_user' do
-        let(:saml_user_attributes) { loa3_user.attributes.merge(loa3_user.identity.attributes) }
-
-        it 'creates an after login job' do
-          allow(SAML::User).to receive(:new).and_return(saml_user)
-          expect { post :saml_callback }.to change(AfterLoginJob.jobs, :size).by(1)
-        end
-      end
-    end
```

Comment:
woah, this spec makes no sense... good catch.

Reply:
I just moved it. There were two `"context 'when not logged in'` blocks